src/UI/TerminalView.re

```diff
@@ -23,14 +23,14 @@ let make =
     (
       ~metrics: EditorMetrics.t,
       ~terminal: Feature_Terminal.terminal,
-      ~editorFont: Service_Font.t,
+      ~font: Service_Font.t,
```

Comment:
Not sure about having the font type be the "main" type of a service. I'm leaning towards services and features not having a `t` type at all, since it seem a bit ambiguous, but rather use `model` for the model, and in this case `font`.

Not a big deal though, just food for though.

Reply:
Seems reasonable - it's not clear what `Service_Font.t` is (is it a service? is it a font?).

Reply:
Updated from `Service_Font.t` -> `Service_Font.font`